test/integration/security/user-group-management-mongo.js

```diff
@@ -0,0 +1,900 @@
+module.exports = TestMesh;
+
+function TestMesh() {
+}
+
+TestMesh.prototype.method1 = function ($happn, options, callback) {
+  options.methodName = 'method1';
+  callback(null, options);
+};
+
+if (global.TESTING_USER_MANAGEMENT) return; // When 'requiring' the module above.
+
+describe(require('../../__fixtures/utils/test_helper').create().testName(__filename, 3), function () {
+
+  this.timeout(120000);
+
+  var expect = require('expect.js');
+  require('chai').should();
+  var mesh;
+  var Mesh = require('../../..');
+
+  var adminClient = new Mesh.MeshClient({secure: true, port: 8003});
+  var test_id = Date.now() + '_' + require('shortid').generate();
+
+  let DB_NAME = 'test-happner-2-user-group-management';
+  let COLL_NAME = 'test-happner-2-user-group-management-coll';
+
+  before('should clear the mongo collection', async() => {
+    let dropMongoDb = require('../../__fixtures/utils/drop-mongo-db');
+    await dropMongoDb(DB_NAME);
+  });
+
+  before(function (done) {
+    global.TESTING_USER_MANAGEMENT = true;
+    mesh = this.mesh = new Mesh();
+    mesh.initialize({
+      name: 'user-management',
+      happn: {
+        secure: true,
+        adminPassword: test_id,
+        port: 8003,
+        services:{
+          data: {
+            config: {
+              autoUpdateDBVersion: true,
+              datastores: [{
+                name: 'mongo',
+                provider:'happn-service-mongo-2',
+                isDefault: true,
+                settings: {
+                  database: DB_NAME,
+                  collection: COLL_NAME
+                }
+              }]
+            }
+          }
+        }
+      },
+      modules: {
+        'TestMesh': {
+          path: __filename
+        }
+      },
+      components: {
+        'TestMesh': {
+          moduleName: 'TestMesh',
+          schema: {
+            exclusive: false,
+            methods: {}
+          }
+        }
+      }
+
+    }, function (err) {
+      if (err) return done(err);
+      mesh.start(function (err) {
+        if (err) {
+          return done(err);
+        }
+        var credentials = {
+          username: '_ADMIN',
+          password: test_id
+        };
+        adminClient.login(credentials).then(function () {
```

Comment:
can just pass done here instead of creating new function

Reply:
done